src/genie/libs/parser/iosxr/tests/ShowRouteAllSummary/cli/equal/golden_output2_expected.py

```diff
@@ -0,0 +1,182 @@
+expected_output = {
+   "vrf": {
+      "default": {
+         "address_family": {
+            "ipv4_unicast": {
```

Comment:
this is value, so I believe we can take it as is from output. Like:
```
expected_output = {
   "vrf": {
      "default": {
         "address_family": {
            "IPv4 Unicast": { # <<<<<

Reply:
Hi Tahigash,

Thanks for your comments... I'll take the variable as it is...

Regards!
Ernesto Wilson